backend/routes/Auth.ts

```diff
@@ -104,4 +106,10 @@ router.get('/RBAC', authenticateWithCookie, (req, res) => {
         });
 });
 
+router.get('/is-first-login', (_req, res, next) => {
```

Comment:
I think `first-login` suits better for a resource name:
```suggestion
router.get('/first-login', (_req, res, next) => {
```

Reply:
OK, changed.